### ISSUE TITLE
Development branch pipeline is failing due to detector library update.

### ISSUE THREAD

**Issue by jgalan** [rest-for-physics/framework#40]
Hi, I found that the upgrade of the detector library to v1.1 at the development branch causes wrong results at pandax-iii MC pipelines.

It is a bit tricky because if the problem is found, in order to solve the issue at the pipeline, the detector library must be increased to version 1.2 and added to the main framework. Since the framework pipeline is running with the "official" library linked.

Plz @nkx could you have a look to check why the pandax-iii MC Pipeline is failing?

**Comment by jgalan** [comment]
By the way, I added the pipeline status at the README.md.

[![pipeline status](https://gitlab.cern.ch/rest-for-physics/framework/badges/development/pipeline.svg)](https://gitlab.cern.ch/rest-for-physics/framework/-/commits/development)

**Comment by nkx111** [comment]
I have found the bug. It is in `TRestMetadata::GetParametersList()`. I found that it did not fully support parameter with units in separate attribute. i.e., the units definition in the rml file of pandax-iii MC: `<parameter name="electricField" value="1000" units="V/cm" />` will be ignored.

I have made the fix in the development branch https://github.com/rest-for-physics/framework/commit/f8838f4f57e64f35e4defede9faded4cbc881061
